app/views/application/_brand_logo.html.erb

```diff
@@ -1,3 +1,3 @@
-<div class="container" style="margin-bottom: 30px">
+<div class="container m-b-30">
```

Comment:
I like to prefix utility/helper classes –like this one– with `u-`, so I would use something like `u-mb-30` (totally opinionated but useful to differentiate utility classes from the rest, ata  glance).